redaxo/src/core/assets/standard.js

```diff
@@ -799,3 +799,21 @@ var handleKeyEvents = function (event) {
         }
     }
 }
+
+function searchfield_init() {  
```

Comment:
`rex_` prefix? haben wir sonst nicht, aber ich denke wenn du hier bewusst eine function haben willst die von anderen mitbenutzt können werden soll, wäre es gut.

Reply:
Denke ich auch.

Und als öffentliche Api frage ich mich, ob man dann vielleicht besser selbst das Element als Parameter übergeben sollte, in dem das Searchfield initialisiert werden soll.
Um keine Doppelinitialisierungen zu erhalten.

Reply:
rex_ ist jetzt hinzugefügt. Ich habe es extra als searchfield benannt weil ich search alleine gerne offen halten würde, falls wir mal eine haben sollten. 
Im Fall von Quicknavi komme ich um eine doppelte Initialisierung nicht herum, da es ein ein anderer trigger ist als rex:ready ... Vielleicht kann @tbaddade da noch was beisteuern.

Reply:
Mir geht es eher darum: Kann es auf einer Seite auch zwei Searchfelder geben?
Wenn du dann dein Quicknavi-Feld selbst initialisiert, indem du die Funktion aufrufst, dann würde eine eventuell woanders schon existierendes Searchfield durch diesen Aufruf erneut mit initialisiert werden, und hätte dann eventuell doppelte Eventhandler.
Daher wäre es vielleicht besser, wenn du dann dein Searchfield aus Quicknavi explizit an die Funktion übergeben würdest, sodass nur das durch deinen Aufruf initialisiert wird.

Reply:
Ich weiß nicht wie. Würde mich da über Hilfe freuen.

Reply:
Ich denke ich habe die Lösung - siehe weiter unten

Reply:
Ich glaube, ich würde es immer noch anders machen. aber ich muss es mir nochmal anschauen. bin gestern nicht mehr dazu gekommen.